CCXDEV-5371 Cloud migration link fixes (#34748)
openshift/openshift-docs: modules/displaying-potential-issues-with-your-cluster.adoc

[id="displaying-potential-issues-with-your-cluster_{context}"]
= Displaying potential issues with your cluster

This section describes how to display the {insights-title} report in the {link-ocm-title}.

> Note that {insights-title} repeatedly analyzes your cluster and shows the latest results. These results can change, for example, if you fix an issue or a new issue has been detected.

Reviewer: ```suggestion
Note that {insights-title} repeatedly analyzes your cluster and shows the latest results. These results can change, for example, if you fix an issue or a new issue is detected.
```

> Write in the simple present tense as much as possible.
